book/src/validator-manager-move.md

```diff
@@ -167,6 +163,7 @@ At the same time, `lighthouse vc` will log:
 INFO Importing keystores via standard HTTP API, count: 1
 INFO Enabled validator                       voting_pubkey: 0xab6e29f1b98fedfca878edce2b471f1b5ee58ee4c3bd216201f98254ef6f6eac40a53d74c8b7da54f51d3e85cacae92f, signing_method: local_keystore
 INFO Modified key_cache saved successfully
+``````
```

Comment:
I think we might be able to remove the extra back ticks here?

Reply:
Thanks, revised